packages/angular_devkit/build_angular/src/differential-loading/update-webpack-config.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
```

Comment:
needs some file-level commenting explaining what's going on, in case the reader hasn't seen the design doc. linking to the design might also be nice